src/Moryx.CommandCenter.Web/src/common/components/Menu/RoutingMenuItem.tsx

```diff
@@ -44,15 +36,33 @@ function RoutingMenuItem(props: MenuItemProps) {
         }
     };
 
-    const isActive = isOpened(location);
+    const isActive = (location: Location): boolean => {
+        // Path has to be equal to be 'active' or must be a sub path (following
+        // After a `/`). Otherwise, with similar entries, multiple list items
+        // Could be highlighted. E.g.: 'Orders' and 'OrdersSimulator' would both
+        // Match the condition of `OrdersSimulator.startsWith(Orders)`.
+        return location.pathname === props.MenuItem.NavPath
+           || (location.pathname.startsWith(props.MenuItem.NavPath)
+                && location.pathname.replace(props.MenuItem.NavPath, "")[0] === "/");
+    };
+
+    const isLocationActive = isActive(location);
 
     return (
-        <ListGroupItem active={isActive} className="menu-item" onClick={(e: React.MouseEvent<HTMLElement>) => handleMenuItemClick(e)}>
-            <Link to={props.MenuItem.NavPath}>
```

Comment:
It is handled by the ListItemButtons click event. Might be better to change that, since the only reason was, that `Link`s would come with blue/purple color which would require custom styling.

Reply:
I don't know but I fell it might even be cleaner this way, since links have a very dedicated purpose which is reflected in their default styling🤔 Since this looks and feels different then a link I would not say it is necessarily better to use a link element. But I leave it up to you

Reply:
It's using a NavLink now to have proper routing.